code/controllers/subsystem/starmap.dm

```diff
@@ -341,16 +341,21 @@ SUBSYSTEM_DEF(starmap)
 	ftl_drive.status_update(message)
 
 /datum/controller/subsystem/starmap/proc/ftl_sound(var/sound) //simple proc to play a sound to the crew aboard the ship, also since I want to use minor_announce for the FTL notice but that doesn't support sound
-	for(var/area/shuttle/ftl/F in world)
-		F << sound
+	var/area/shuttle/ftl/cargo/mining/M
+	if(M.z == ZLEVEL_STATION)	//check if the FOB shuttle is with the main ship
```

Comment:
M is null here, it hasn't been assigned a value. Loop through the areas in GLOB.sortedAreas first then get a turf per iteration and check it's z